empiric-package/empiric/core/entry.py

```diff
@@ -4,13 +4,25 @@
 
 from empiric.core.utils import felt_to_str, str_to_felt
 
-
-class Entry:
+class BaseEntry:
+    # Make this the generic
```

Comment:
Should this be generic or should there be a base that is inherited by the other types?

Reply:
I think we can have both, but for typing it should be a generic. E.g. fetch returns a list of Entry's, where each Entry is one of GenericEntry, SpotEntry or FutureEntry. There may be a more elegant way to handle that but I'm thinking this would be useful for instances where we have *some* kind of Entry and don't want to specify which kind